src/main/java/thePackmaster/cards/overwhelmingpack/Withstand.java

```diff
@@ -1,26 +1,42 @@
 package thePackmaster.cards.overwhelmingpack;
 
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
 import com.megacrit.cardcrawl.actions.defect.ReinforcedBodyAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
 import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
 import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.util.Wiz;
 
 import static thePackmaster.SpireAnniversary5Mod.makeID;
 
 public class Withstand extends AbstractOverwhelmingCard {
     public final static String ID = makeID("Withstand");
 
     public Withstand() {
-        super(ID, -1, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        super(ID, 2, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
 
-        this.baseBlock = this.block = 6;
-        this.selfRetain = true;
+        this.baseBlock = this.block = 4;
+        this.baseMagicNumber = this.magicNumber = 1;
     }
 
     public void use(AbstractPlayer p, AbstractMonster m) {
-        this.addToBot(new ReinforcedBodyAction(p, this.block, this.freeToPlayOnce, this.energyOnUse));
+        int blck = this.block, threshold = this.magicNumber;
+        addToBot(new AbstractGameAction() {
+            @Override
+            public void update() {
+                this.isDone = true;
+                for (AbstractCard c : p.hand.group) {
+                    if (Wiz.getLogicalCardCost(c) <= threshold) {
```

Comment:
Unplayable cards are treated as 0 cost for that method, up to you if you want that.

Reply:
(I do)